src/sentry/conf/server.py

```diff
@@ -1676,6 +1679,11 @@ def get_sentry_sdk_config():
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default", "topic": KAFKA_INGEST_TRANSACTIONS},
 }
 
+# Kafka topics configuration for eventstream
+SENTRY_EVENTSTREAM_DEFAULT_TOPIC = KAFKA_EVENTS
+# Mapping between event type and topic. Those not defined here fallback to deault.
+SENTRY_EVENTSTREAM_TOPICS = {}
```

Comment:
Should these be just part of `SENTRY_EVENTSTREAM_OPTIONS` (which gets provided as `**options` to `EventStream`, via [this block](https://github.com/getsentry/sentry/blob/4682ba84362c7c639b83f9a3939b5185b357fb10/src/sentry/eventstream/__init__.py#L9-L11))?

This seems very coupled to `KafkaEventStream` and none of the other backend implementations (or code outside of the `KafkaEventStream` instance itself), so I'm not sure specifying it at this global settings level makes sense.

Reply:
re `Should these be just part of SENTRY_EVENTSTREAM_OPTIONS`. 
My understanding about those options was that they represent service delegator options like `backend_base`, `backends`, `selector_func` etc.  Is that the case? if not sure, I will move them there.

Reply:
They can be used for that, but they can also get provided to the backend itself ([the service delegator will also proxy these through](https://github.com/getsentry/sentry/blob/4682ba84362c7c639b83f9a3939b5185b357fb10/src/sentry/utils/services.py#L240-L245), but this is not clearly documented).

The Redis TSDB is a decent (though somewhat convoluted) example: https://github.com/getsentry/sentry/blob/a5880be7602458a706b1242498166350e1c445a7/src/sentry/tsdb/redis.py#L113-L118